### DISCUSSION TITLE
Custom elements

### DISCUSSION THREAD

**Discussion by cpsievert** [Knio/dominate#164] [Q&A] [answered]
I may be missing something, but there doesn't seem to be a way to construct [custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) (or [JSX](https://reactjs.org/docs/introducing-jsx.html))? If not, maybe it could added via an API that looks something like this?

```python
import dominate
dominate.tags.tag('custom-element', 'children', foo='bar')
```

```html
<custom-element foo='bar'> children </custom-element>
```

**Comment by Knio** [comment, marked as answer]
See #151
